### PR TITLE
Deck voting

### PR DESCRIPTION
**Pull request by alexmiddeleer**

### CODE REVIEW

**Comment by lily-pytel** [comment]
Oh man oh man, the most anticipated feature

File: next/components/deck-export.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ DeckExport.propTypes = {
       id: PropTypes.number,
       name: PropTypes.string
     }),
-    deckPath: PropTypes.shape({
+    deckPower: PropTypes.shape({
```
Comment (lily-pytel) [inline next/components/deck-export.js]: This again?!

Comment (alexmiddeleer) [reply next/components/deck-export.js]: I don't even know. This is mgh's weirdest issue.

**Comment by lily-pytel** [comment]
image

Is there not supposed to be an upvote symbol there that takes you to the login page?

Also when I logged in and went to this page (http://localhost:3001/deck?id=3), I got this:

image

I know it's because that deck has no author and that case is impossible but I think it's worth putting a safeguard there anyway

Then I tried to go to the cats deck and vote but got:

image

Console:

image

Network was more helpful: 

image

In conclusion, I sadly can't vote yet! 😆

**Comment by alexmiddeleer** [comment]
@lily-pytel Thanks for the review! I think that last error was just that you needed to restart the express container because of the query whitelist. But definitely some good catches.

**Comment by alexmiddeleer** [comment]
@lily-pytel Also, mind sanity-checking the last commits or if you have time, trying to hack it?

**Comment by lily-pytel** [comment]
@alexmiddeleer yep, looking now.

> I think that last error was just that you needed to restart the express container because of the query whitelist. But definitely some good catches.

That would be strange because I basically destroy all containers and rebuild between every branch or database change or PR or almost every day, really
